llvm/test/CodeGen/X86/code_placement_ext_tsp_size.ll

```diff
@@ -0,0 +1,131 @@
+; RUN: llc -mcpu=corei7 -mtriple=x86_64-linux -apply-ext-tsp-for-size=true  < %s | FileCheck %s
```

Comment:
Can we name `CHECK` more explicitly instead of `CHECK` and `CHECK2`? e.g.,
```suggestion
; RUN: llc -mcpu=corei7 -mtriple=x86_64-linux -apply-ext-tsp-for-size=true  < %s | FileCheck %s --check-prefix=CHECK-SIZE
```